scripts/mkcloud

```diff
@@ -720,6 +729,35 @@ function instnodes
     safely onadmin post_allocate
 }
 
+function deployexternalceph
+{
+    if [[ $want_external_ceph == 0 ]] ; then
+        echo "\$want_external_ceph not set, skipping"
+        return
+    fi
+    if [[ $external_ceph_nodes == 0 ]] ; then
+        echo "\$external_ceph_nodes = 0, can't build a ceph cluster of 0 nodes"
+        return
+    fi
+    # First we want to use setuplonelynodes to set up some nodes
+    # for the external cluster, we'll use externalcephnodes as
+    # nodenumberlonelynode's number
+    local old_lonelynodes
+    old_lonelynodes=$nodenumberlonelynode
+    nodenumberlonelynode=$externalcephnodes
+    echo "== Running setuplonelynodes step with $externalcephnodes nodes =="
+    safely setuplonelynodes
+    echo "== Running crowbar_register step =="
+    safely crowbar_register
```

Comment:
might be preferrable ot leave out the ~10 lines above and instead document that users should do
```
export nodenumberlonelynode=...
mkcloud ... setuplonelynodes crowbar_register deployexternalceph
```
which allows to do custom things in between, debug and re-run parts in case of issues.

Reply:
@bmwiedemann Thanks, that makes much more sense. I've mentioned you have to run the 2 previous steps in the deployexternalceph's usage help text. Other then in the commit message itself, is there anywhere we store other such notes for users? ie in this or other repo's I should add this to?